src/fs_utils.rs

```diff
@@ -96,6 +96,12 @@ pub fn filter_events(
             continue;
         }
 
+        // Ignore ordinary files peer to config.toml. This assumes all other files we care
+        // about are nested more deeply than config.toml or are directories peer to config.toml.
+        if path != config_path && path.is_file() && path.parent() == config_path.parent() {
```

Comment:
I added an additional condition here, `path.is_file()`. Presumably we could be watching something like `sass/` peer to `config.toml`, and a user could remove the directory for some reason and then add it back later, all while `zola serve` is watching. We don't want to skip such changes, so I filter on regular files, not directories.